src/main/java/org/opentripplanner/middleware/otp/response/OtpResponse.java

```diff
@@ -11,7 +11,7 @@
  * Pare down version of class original produced for OpenTripPlanner.
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class Response {
+public class OtpResponse {
```

Comment:
I'm pretty sure @br648 named this based on the version of the file from OpenTripPlanner. So, the javadoc should at least have a qualified class name and/or URL to the file in the OTP codebase if we're changing its name.

Reply:
Added in 6fd00a976c063c5b46e7254a995fbbb628b90445